lib/systems/platforms.nix

```diff
@@ -521,5 +521,7 @@ rec {
 
     else if platform.parsed.cpu == lib.systems.parse.cpuTypes.powerpc64le then powernv
 
-    else pc;
+    else if platform.isAvr then pc   # not sure this is right, but preserves existing behavior
+
+    else abort ("could not detect platform: " + (builtins.toJSON platform.parsed.cpu));
```

Comment:
Any reason you didn't use `throw`?

Reply:
Because I am too new to nix-lang and didn't know about it!  Fixed.